satpy/modifiers/angles.py

```diff
@@ -231,11 +269,52 @@ def _sanitize_observer_look_args(*args):
         elif isinstance(arg, (float, np.float64, np.float32)):
             # round floating point numbers to nearest tenth
             new_args.append(round(arg, 1))
+        elif _is_chunk_tuple(arg) and _chunks_are_irregular(arg):
+            new_chunks = _regular_chunks_from_irregular_chunks(arg)
+            new_args.append(new_chunks)
```

Comment:
I suppose it's not possible to use the more pythonic try/except instead of checking the regularity of the chunks preemptivly?

Reply:
I'd prefer not to given the number of errors/exceptions that could arise from calling in to `to_zarr` that may or may not be related to irregular chunking.

Reply:
ok